.eslintrc.json

```diff
@@ -1,3 +1,47 @@
 {
-  "extends": ["next/core-web-vitals", "prettier"]
+  "extends": ["next/core-web-vitals", "prettier"],
+  "parserOptions": {
+    "sourceType": "module"
+  },
+  "env": { "es6": true },
+  "plugins": ["simple-import-sort"],
+  "rules": {
+    // Else Eslint will throw an error for `!!raw-loader!` imports
+    "import/no-webpack-loader-syntax": [0],
+    "simple-import-sort/imports": [
+      "error",
+      {
+        "groups": [
+          // Node.js builtins.
+          [
+            "^(assert|buffer|child_process|cluster|console|constants|crypto|dgram|dns|domain|events|fs|http|https|module|net|os|path|punycode|querystring|readline|repl|stream|string_decoder|sys|timers|tls|tty|url|util|vm|zlib|freelist|v8|process|async_hooks|http2|perf_hooks)(/.*|$)"
+          ],
+          // Packages. `react` related packages come first.
+          // Also, put `react-icons` in sorting order not with `react`
+          ["^react(?!-.)$", "^\\w", "^@\\w"],
+          // The Chakra theme directory if imported to story file or other places
+          ["^(\\.\\./)+@chakra-ui/gatsby-plugin/theme"],
+          // From the `types` directory.
+          ["^(\\.\\./)+types"],
```

Comment:
Oh that would be cool, yes!

Reply:
Yes, that would be great... please note though, over on #72 we have scripts that are being run from `ts-node` which I was unable to get working using the `@/` shortcut imports and had to use relative paths instead using `../` syntax. If you know a way around that I'm all ears, otherwise just be aware that we may need to ignore some folders from that rule.

Reply:
@wackerow can you clarify what you are referring to in that PR? It looks like the changes in the PR look incomplete; not all the files seem to be there to view. And I can't checkout the PR locally to test it out since it's closed.

Reply:
Oops, sorry @TylerAPfledderer, wrong PR... https://github.com/ethereum/ethereum-org-fork/pull/73 (but now merged into `dev`)... This imported scripts in `src/scripts/crowdin`, which are executed as a node script. When I used the `@` shortcut for the imports it broke the script execution. The main script here is `src/scripts/crowdin/getCrowdinContributors.ts` if you want to check it out.

Quick example, if I change one of the imports in that file:
<img width="922" alt="image" src="https://github.com/ethereum/ethereum-org-fork/assets/54227730/128d5d67-6eae-45e8-8cc5-0d897c5d2440">

Doesn't look like `./src/scripts` is excluded in the tsconfig, and it's included with `./src/**/*`.

Reply:
With your example, shouldn't it be `@/scripts/crowdin/reports/getTranslationCostsReports`?

Reply:
Okay, goofing this up all over... yes, it would... and I was also accidentally in the canonical repo for that screenshot... let me start over:

<img width="574" alt="image" src="https://github.com/ethereum/ethereum-org-fork/assets/54227730/42e9e34d-ee32-47f0-b300-05341b1f2b22">

Attempting to change one of these, and then running `yarn crowdin-contributors` I get:

```
/ethereum-org-fork [pr/96]  yarn crowdin-contributors
yarn run v1.22.19
$ ts-node -O '{ "module": "commonjs" }' src/scripts/crowdin/getCrowdinContributors.ts
Error: Cannot find module '@/scripts/markdownChecker'
Require stack:
- /ethereum-org-fork/src/scripts/crowdin/getCrowdinContributors.ts
    at Function.Module._resolveFilename (node:internal/modules/cjs/loader:1048:15)
    at Function.Module._resolveFilename.sharedData.moduleResolveFilenameHook.installedValue [as _resolveFilename] (/ethereum-org-fork/node_modules/@cspotcode/source-map-support/source-map-support.js:811:30)
    at Function.Module._load (node:internal/modules/cjs/loader:901:27)
    at Module.require (node:internal/modules/cjs/loader:1115:19)
    at require (node:internal/modules/helpers:119:18)
    at Object.<anonymous> (/ethereum-org-fork/src/scripts/crowdin/getCrowdinContributors.ts:1:1)
    at Module._compile (node:internal/modules/cjs/loader:1233:14)
    at Module.m._compile (/ethereum-org-fork/node_modules/ts-node/src/index.ts:1618:23)
    at Module._extensions..js (node:internal/modules/cjs/loader:1287:10)
    at Object.require.extensions.<computed> [as .ts] (/ethereum-org-fork/node_modules/ts-node/src/index.ts:1621:12) {
  code: 'MODULE_NOT_FOUND',
  requireStack: [
    '/ethereum-org-fork/src/scripts/crowdin/getCrowdinContributors.ts'
  ]
}
error Command failed with exit code 1.
info Visit https://yarnpkg.com/en/docs/cli/run for documentation about this command.
```

Reply:
Ahh I see.

Hmm I couldn't find anything on how or why this is happening. For this example, it would certainly make more sense to just do the relative path for that import instead of the alias.

Otherwise, I found the use of [tsx](https://github.com/esbuild-kit/tsx) instead of `ts-node` could handle this case without declaring `module: commonJS`.